### DISCUSSION TITLE
Add support for Databend to act as MySQL replica

### DISCUSSION THREAD

**Discussion by itziklavon** [databendlabs/databend#6403] [Ideas]
Clickhouse has this database engine:
MaterializedMySQL

currently i'm using clickhouse, and my mysql instance is on AWS RDS
however, this engine does not support AWS RDS for replication

so, the feature is couple of things actually
1. Add support for databend to act as mysql replica
2. Add support to allow replica of cloud services(AWS RDS included)

Thanks!

**Comment by sundy-li** [comment]
Ok, since the feature of `MaterializedMySQL` engine in `ClickHouse` is also from our team, we do have a plan to support it in Databend in the future.
